[2022-06-29 20:12:14,505] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-28T21:12:00+00:00 [queued]>
[2022-06-29 20:12:14,516] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-28T21:12:00+00:00 [queued]>
[2022-06-29 20:12:14,517] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 20:12:14,517] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-29 20:12:14,517] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 20:12:14,556] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-28 21:12:00+00:00
[2022-06-29 20:12:14,560] {standard_task_runner.py:52} INFO - Started process 2614 to run task
[2022-06-29 20:12:14,563] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-28T21:12:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/CarPriceDag.py', '--cfg-path', '/tmp/tmp51al1h6i', '--error-file', '/tmp/tmp9eb998b2']
[2022-06-29 20:12:14,563] {standard_task_runner.py:80} INFO - Job 278: Subtask save_data_task
[2022-06-29 20:12:14,620] {task_command.py:369} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-28T21:12:00+00:00 [running]> on host DESKTOP-9B44JSS.localdomain
[2022-06-29 20:12:14,742] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-28T21:12:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-28T21:12:00+00:00
[2022-06-29 20:12:14,972] {python.py:173} INFO - Done. Returned value was: None
[2022-06-29 20:12:14,978] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220628T211200, start_date=20220629T181214, end_date=20220629T181214
[2022-06-29 20:12:15,018] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-29 20:12:15,060] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
